src/main/java/wordle/domain/Letter.java

```diff
@@ -0,0 +1,48 @@
+package wordle.domain;
+
+import java.util.Objects;
+
+public class Letter {
+    private static final char SMALL_A = 'a';
+    private static final char SMALL_Z = 'z';
+
+    private final String value;
+
+    public Letter(final String value) {
+        validate(value);
+        this.value = value;
+    }
+
+    private void validate(final String value) {
+        for (char charValue : value.toCharArray()) {
+            if (SMALL_Z < charValue || charValue < SMALL_A) {
+                throw new IllegalArgumentException("글자는 영어 소문자만 입력가능합니다.");
+            }
+        }
+    }
```

Comment:
개인적인 의견이지만 `SMALL_Z < charValue || charValue < SMALL_A` 보단 `charValue < SMALL_A || SMALL_Z < charValue`가 더 이해하기 쉬운 것 같은데 가비의 생각은 어때?

추가로 아래와 같이 구현한다면 1 depth로 충분할 것 같아

```suggestion
    private void validate(final String value) {
        if (value.chars().anyMatch(this::isInvalidRange)) {
            throw new IllegalArgumentException("글자는 영어 소문자만 입력가능합니다.");
        }
    }

    private boolean isInvalidRange(final int charValue) {
        return charValue < SMALL_A || SMALL_Z < charValue;
    }
```

Reply:
사실 쓰면서 알게모르게 좀 찝찝했는데.. chars() 메서드는 처음알았네

오늘도 배우고간담

Reply:
경계값 범위는

1 < x < 5  이런느낌으로 검증대상의 범위를 보여주는 게 읽기 좋다는 생각이었는데
오히려 모아두는게 더 나은 것 같기도 하고...?